CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -184,44 +191,37 @@ public protocol TestingImageDataSource: AnyObject {
         self.videoFeed.requestCameraAccess(permissionDelegate: self)
     }
     
-    func setVideoOrientation() {
-        if ScanBaseViewController.isPadAndFormsheet {
-            self.previewView?.videoOrientation = AVCaptureVideoOrientation(rawValue: UIWindow.interfaceOrientation.rawValue)
-            self.videoFeed.videoOrientation = self.previewView?.videoOrientation
-        } else {
-            UIDevice.current.setValue(UIDeviceOrientation.portrait.rawValue, forKey: "orientation")
-            self.previewView?.videoOrientation = .portrait
-            self.videoFeed.videoOrientation = .portrait
-        }
-    }
-    
     public func setupOnViewDidLoad(regionOfInterestLabel: UIView, blurView: BlurView, previewView: PreviewView, cornerView: CornerView?, debugImageView: UIImageView?, torchLevel: Float?) {
         
         self.regionOfInterestLabel = regionOfInterestLabel
         self.blurView = blurView
         self.previewView = previewView
         self.debugImageView = debugImageView
         self.cornerView = cornerView
+        ScanBaseViewController.isPadAndFormsheet = UIDevice.current.userInterfaceIdiom == .pad && self.modalPresentationStyle == .formSheet
```

Comment:
can we make `isPadAndFormsheet` a member variable instead of a static? You can pass it into the setup function for the videoFeed if needed, but having this as a static can be problematic because it's specific to an individual ViewController and not something that is global

Reply:
i made it static to use in the static supportedInterfaces function

Reply:
ugg ok ignore this for now

Reply:
:-|